src/main/java/pl/karolpat/service/ParkingMeterService.java

```diff
@@ -0,0 +1,90 @@
+package pl.karolpat.service;
+
+import java.util.List;
+import java.util.Map;
+
+import pl.karolpat.entity.ParkingMeter;
+import pl.karolpat.entity.User;
+
+/**
+ * @author karolpat
+ *
+ */
+public interface ParkingMeterService {
+
+	/**
+	 * Saves ParkingMeter instance to the database.
+	 * 
+	 * @param PM
+	 *            ParkingMeter instance to be saved to the database.
+	 * @return ParkingMeter instance that is being saved.
+	 */
+	ParkingMeter save(ParkingMeter PM);
```

Comment:
zgodnie z nazewnictwem Javy - czy zmienna PM jest ok?

Reply:
Jak najbardziej nie jest ok :). Nie wiem co we mnie wstąpiło